parser.y

```diff
@@ -4755,23 +4755,36 @@ LikeEscapeOpt:
 Field:
 	'*'
 	{
-		$$ = &ast.SelectField{WildCard: &ast.WildCardField{}}
+		$$ = &ast.SelectField{
+			WildCard: &ast.WildCardField{},
+			Offset:   parser.startOffset(&yyS[yypt]),
```

Comment:
The offset field has been initialized here:

https://github.com/pingcap/parser/blob/12de604100a41a10651cd9083a16565284813c40/parser.y#L4845-L4851

What's the purpose to make this change? Are there any tests or use cases?

Reply:
Hmm, if this is being populated I'm not sure why I needed to change parser.y at all. I'll need to investigate why it wasn't working for me.